pkg/heartbeat/osname_linux.go

```diff
@@ -0,0 +1,33 @@
+//go:build linux
+
+package heartbeat
+
+import (
+	"syscall"
+
+	"github.com/wakatime/wakatime-cli/pkg/log"
+)
+
+func osName() string {
```

Comment:
This function is not related to heartbeat so let's move to another package. I think a good descriptor could be `system` (any suggestion?). The it would look like `system.OsName()`.